packages/lesswrong/components/comments/CommentsListSection.jsx

```diff
@@ -80,6 +80,8 @@ class CommentsListSection extends Component {
     const { anchorEl, highlightDate } = this.state
     const { CommentsListMeta } = Components
     const suggestedHighlightDates = [moment().subtract(1, 'day'), moment().subtract(1, 'week'), moment().subtract(1, 'month'), moment().subtract(1, 'year')]
+    const newLimit = commentCount + (loadMoreCount || commentCount)
+    console.log("newLimit: ", newLimit)
```

Comment:
```suggestion
    console.log("newLimit: ", newLimit)
```

Reply:
Huh, apparently we no longer run lint on tests. I should fix that.

Reply:
Fixed.